dsp/modules/sentence_vectorizer.py

```diff
@@ -247,4 +252,65 @@ def __call__(self, inp_examples: List["Example"]) -> np.ndarray:
         texts_to_vectorize = self._extract_text_from_examples(inp_examples)
         embeddings = self._model.embed(texts_to_vectorize, batch_size=self._batch_size, parallel=self._parallel)
 
-        return np.array([embedding.tolist() for embedding in embeddings], dtype=np.float32)
\ No newline at end of file
+        return np.array([embedding.tolist() for embedding in embeddings], dtype=np.float32)
+
+
+def get_premai_api_key(api_key: Optional[str] = None) -> str:
```

Comment:
https://github.com/stanfordnlp/dspy/blob/c6219ca462baab32cddc671d978ad3190f31cec2/dsp/modules/premai.py#L38

We can use this function here by importing it.

Reply:
Thanks for pointing it out, fixed :)